sections/editing.include

```diff
@@ -1667,14 +1668,24 @@
   If specified, the value must be a single printable character:
   typically a string one Unicode code point in length.
   
-  <p class="warning">Character references such as <code>accesskey="&#65"</code> are not valid values, 
-  because they will cause no shortcut to be assigned in many browsers. 
-  However certain printable characters that may represent more than one Unicode code point, such as 
-  <code>accesskey="श्र"</code> are valid.</p>
+  <div class="warning">
+    Composed characters in
+    <a href="https://www.w3.org/International/questions/qa-html-css-normalization">Normalised Form D</a>,
+    such as <code>accesskey="&amp;#x006E;&amp;#x0303;"</code>
+    to assign <samp>ñ</samp> as a shortcut, are not valid values,
+    and in many browsers no shortcut will be assigned. Characters in
+    <a href="https://www.w3.org/International/questions/qa-html-css-normalization">Normalised Form C</a>
+    are valid values. Printable characters that may represent more than one Unicode code point, such as 
+    <code>accesskey="श्र"</code>, are valid values.
   
```

Comment:
Indic sequences that are rendered as a single glyph seem to be OK. It is possible that something is normalising them before setting up the event - that certainly *seemed* to be the case on Mac. Although the examples you gave me are the only case I could test and find that something worked. But then, the handling of latin characters has other apparently undocumented magic driving it, as seen in testing for case-sensitivity across different alphabets.

Outlawing sequences of characters that are not a single printable character is because browsers which would recognise the individual characters - including e.g. ñ, fail to recognise the `accesskey` attribute at all in such cases, which is even more broken :(